cuML/CMakeLists.txt

```diff
@@ -161,20 +169,25 @@ add_dependencies(gpufaisslib faiss)
 set_property(TARGET faisslib PROPERTY IMPORTED_LOCATION ${FAISS_DIR}/libfaiss.a)
 set_property(TARGET gpufaisslib PROPERTY IMPORTED_LOCATION ${FAISS_DIR}/gpu/libgpufaiss.a)
 
+
 include_directories(src
-	    ${CMAKE_CURRENT_BINARY_DIR}/faiss/include
+  	    ${CMAKE_CURRENT_BINARY_DIR}/faiss/include
 	    ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}
         ${GTEST_DIR}/include
         ${MLPRIMS_DIR}/src
         ${MLPRIMS_DIR}/test
         ${CUTLASS_DIR}
         ${CUB_DIR}
+        ${ZLIB_INCLUDE_DIRS$},
+        # TODO : Fix the path
+        /home/mehdia/miniconda3/envs/cuml-0128/include
         ${ZLIB_INCLUDE_DIRS$})
 
 add_subdirectory(${GTEST_DIR} ${PROJECT_BINARY_DIR}/googletest)
 
 # Append source file in recursive manner, append header files to target for work with them in IDE
 file(GLOB_RECURSE ml_prims_header "${MLPRIMS_DIR}/src/*.h" "${MLPRIMS_DIR}/src/*.hpp")
+file(GLOB_RECURSE magma_prims_test_sources "${MLPRIMS_DIR}/test/hmm/magma/*.cu")
```

Comment:
If it is part of ml-prims, I think these tests should be moved to cmake file of ml-prims folder.

Reply:
I still need to work on that